patient/rotary.py

```diff
@@ -2,9 +2,14 @@
 
 import sys
 import time
-import pigpio
 import threading
 
+try:
+    import pigpio
+except ImportError:
+    # in case we're loading this just for the MockRotary
+    pass
```

Comment:
Could you put a unittest.mock.Mock() or MagicMock() here, I wonder?

Reply:
I'll play with this later, as I need to hook it up with the real rotary when running live. :)

Reply:
I don't know about unitest.mock, but it sounds like I want its moral equivalent. I'm going to let you organize that code as needed. For triggering alarms, I need to have some object that will tell me the current alarm thresholds.